lib/atlas/scaler/graph_persistor.rb

```diff
@@ -0,0 +1,26 @@
+module Atlas
+  class Scaler::GraphPersistor
+    def initialize(dataset, path)
+      @dataset = dataset
+      @graph   = GraphBuilder.build
+      @path    = path
+    end
+
+    def persist_graph
```

Comment:
If the class is apparent than you know what `execute!` does. You mean when you see it called on an object whose class is not immediately apparent to you?
Ok, I see your point. `persist!` sounds good :smile: